package.json

```diff
@@ -20,7 +20,8 @@
     "deploy:prod": "NODE_ENV=production sls deploy --stage production",
     "remove-stack:dev": "sls remove --stage development",
     "zip-release": "zip -r release.zip .serverless serverless.yaml package.json package-lock.json",
-    "setup": "node scripts/setup.js"
+    "setup": "node scripts/setup/setup.js",
+    "setup-file": "node scripts/setup/createSetupFile.js"
```

Comment:
Is this command being run? I don't see any reference to this command or createSetupFile.js being used in any other way